source/02_algo/model_based_rl_eng.rst

```diff
@@ -0,0 +1,147 @@
+Model-Based Reinforcement Learning
+######################################
+
+
+Model-Based Reinforcement Learning (Model-Based RL) is an important branch of reinforcement learning. The agent learns an environmental dynamics model through information interacting with the environment, and then uses the model to generate data to optimize action strategies, or use the model for planning.
+Model-Based RL methods first let the agent learn the environment dynamics model from the data obtained by interacting with the environment, and then use the environment model to generate a large number of virtual samples, thereby reducing the number of interactions with the real environment. Therefore, the sample utilization efficiency can be naturally improved.
```

Comment:
The Model-Based RL method first learns a **_dynamics_** model from the data obtained **_by interacting with the environment_** and then uses the dynamics model to generate a large number of _**simulated samples**_. _**In this way, the number of interactions with the real environment will be reduced, or in other words, the sample efficiency can be greatly improved.**_